scss/titles.scss

```diff
@@ -0,0 +1,16 @@
+.title {
+  color: $black;
+  font-size: $title-size;
+  height: 220px;
+  margin-left: 80px;
+
+  &::before{
+    content: url(assets/triangle.svg);
+    display: block;
+    position: relative;
+    right: 78px;
+    top: 80px;
```

Comment:
By setting the correct height and width you would not need to use a fixed top value. 😬